xsd/netex_part_1/part1_ifopt/netex_ifopt_site_support.xsd

```diff
@@ -128,12 +128,12 @@ Rail transport, Roads and Road transport
 		</xsd:annotation>
 		<xsd:restriction base="SiteIdType"/>
 	</xsd:simpleType>
-	<xsd:element name="ServiceSiteRef" type="ServiceSIteRefStructure" substitutionGroup="SiteRef">
+	<xsd:element name="ServiceSiteRef" type="ServiceSiteRefStructure" substitutionGroup="SiteRef">
 		<xsd:annotation>
 			<xsd:documentation>Reference to a SERVICE SITE.</xsd:documentation>
 		</xsd:annotation>
 	</xsd:element>
-	<xsd:complexType name="ServiceSIteRefStructure">
+	<xsd:complexType name="ServiceSiteRefStructure">
 		<xsd:annotation>
 			<xsd:documentation>Type for identifier of a SERVICE SITE.</xsd:documentation>
```

Comment:
Should documentation be "Type for reference to a SERVICE SITE" ?

Reply:
There are many documentation errors and bad copy/pastes like this. I think this should be a joined effort to fix them.